common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -195,10 +196,10 @@ def test_masquerade_hidden_content_past_due(self):
                 extra_context=dict(specific_masquerade=True),
             )
             self.assertIn("seq_module.html", html)
-            self.assertIn(
-                "'banner_text': u'Because the due date has passed, "
-                "this assignment is hidden from the learner.'",
-                html
+            html = self.get_context_dict_from_string(html)
```

Comment:
This should not have '/n *spaces*' in python3. This will pass the test but not fix the actual bug.

Reply:
python2
```
u"(('seq_module.html',\n  {'ajax_url': 'i4x://edX/xml_test_course/sequential/6/xmodule_handler',\n   'banner_text': None,\n   'element_id': u'i4x-edX-xml_test_course-sequential-6',\n   'gated_content': {'gated': True,\n                     'gated_section_name': u'sequence_1_2',\n                     'prereq_section_name': 'PrereqSectionName',\n                     'prereq_url': 'PrereqUrl'},\n   'item_id': u'i4x://edX/xml_test_course/sequential/6',\n   'items': [],\n   'next_url': 'NextSequential',\n   'position': 1,\n   'prev_url': 'PrevSequential',\n   'save_position': True,\n   'show_completion': True,\n   'tag': 'sequential'}),\n {})"
```

python3
```
"(('seq_module.html',\n  {'ajax_url': 'i4x://edX/xml_test_course/sequential/6/xmodule_handler',\n   'banner_text': None,\n   'element_id': 'i4x-edX-xml_test_course-sequential-6',\n   'gated_content': {'gated': True,\n                     'gated_section_name': 'sequence_1_2',\n                     'prereq_section_name': 'PrereqSectionName',\n                     'prereq_url': 'PrereqUrl'},\n   'item_id': 'i4x://edX/xml_test_course/sequential/6',\n   'items': [],\n   'next_url': 'NextSequential',\n   'position': 1,\n   'prev_url': 'PrevSequential',\n   'save_position': True,\n   'show_completion': True,\n   'tag': 'sequential'}),\n {})"